source/includes/fact-mongodump-mongorestore-use-cases.rst

```diff
@@ -0,0 +1,8 @@
+:binary:`~bin.mongoimport` and :binary:`~bin.mongoexport` are not
+intended for the following use cases:
+
+- backing up and restoring data
+- sending data from a MongoDB instance to another instance and then back
```

Comment:
When I said round tripping I didn't mean it had to go to an intermediate instance, just that the data is meant to go back to a mongoDB instance. Definitition is here: "process of converting data from one format or source to another and back"

Reply:
just made the change!